### PR TITLE
Created and add the EMCal single channel calibration

### PR DESCRIPTION
**Pull request by dhruvdixit**
A new task was written for the single channel calibration and it has been added to the EMCal Correction Framework. By default, the task is disabled and currently only calibrates runs from LHC15o. Merge is needed so that the version is available on the grid for testing.

### CODE REVIEW

File: PWG/EMCAL/config/AliEmcalCorrectionConfiguration.yaml [hunk 1]
```diff
@@ -63,6 +63,13 @@ CellEnergy:                                         # Cell Energy correction com
     customRecalibFilePath: ""                       # Full path including .root file for custom recalibration object
     cellsNames:                                     # Names of the cells input objects which should be attached to the correction
         - defaultCells                              # This object is defined above in the cells section of the input objects
+CellSingleChannelCalibration:                       # Cell Single Channel Energy correction component
+    enabled: false                                  # Whether to enable the task
+    createHistos: false                             # Whether the task should create output histograms
+    enableNewTempCalib: false                       # Whether the new temperature calibration parmeters should be used (available for Run1 and Run2)
```
Comment (raymondEhlers) [inline PWG/EMCAL/config/AliEmcalCorrectionConfiguration.yaml]: Is this option used / needed? I don't see any other references to it

**Comment by raymondEhlers** [comment]
+test

File: PWG/EMCAL/config/AliEmcalCorrectionConfiguration.yaml (hunk 1 above)
Comment (dhruvdixit) [reply PWG/EMCAL/config/AliEmcalCorrectionConfiguration.yaml]: CellSingleChannelCalibration is the correction component for the Single Channel calibration. Locally, I have added and enabled it in  PWG/EMCAL/config/EMCalSampleConfig.yaml for testing. If you are referring to enableNewTempCalib, then yes, that should be removed. And yes, I think most of the commit can be squashed. I will do that.

Comment (raymondEhlers) [reply PWG/EMCAL/config/AliEmcalCorrectionConfiguration.yaml]: Yeah, I was just referring to the `enableNewTempCalib` option. The comment ended up highlighting more than I expected. Thanks!

**Comment by FriederikeBock** [comment]
Hi Dhruv,
from your description I am a bit worried. "By default this task is enabled." Could we for now maybe change the default such that it is disabled and the analyzer has to actively enable it in his/her jaml file and similarly for the tender?
Regards,
Friederike

**Comment by FriederikeBock** [comment]
@dhruvdixit  could you have a look where this stands? Do we still wanna merge this. If so please rebase and solve merge conflicts.
Regards, Fredi

**Comment by dhruvdixit** [comment]
@FriederikeBock My local AliPhysics broke during the rebase, due to an outdated aliBuild and python so I am updating it and then we can resolve the merge conflicts.

**Comment by dhruvdixit** [comment]
I think we should be good to merge.

**Pull request closed without merging**